README.md

```diff
@@ -1,5 +1,6 @@
+<!-- Begin exclude from NuGet readme. -->
```

Comment:
Would it not have been just easier to make two separate files 🤔

Reply:
Perhaps, but we'd 100% forget to update them both (particularly because the GitHub-facing one would be the only one us maintainers see)